autobahn/wamp_session.ipp

```diff
@@ -710,13 +710,15 @@ void wamp_session<IStream, OStream>::process_invocation(
 
     auto procedure_itr = m_procedures.find(registration_id);
     if (procedure_itr != m_procedures.end()) {
+        wamp_invocation invocation = std::make_shared<wamp_invocation_impl>();
+        invocation->set_request_id(request_id);
+
         if (message[3].type != msgpack::type::MAP) {
             throw protocol_error("INVOCATION.Details must be a map");
+        }else{
```

Comment:
`else` (and all the other stuff, like `if`, `catch` and whatnot) is surrounded by spaces in other places in AutobahnCpp. For consistency it would be nice to stick to that.